src/options.html

```diff
@@ -4,7 +4,7 @@
 		<meta charset="utf-8">
 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-		<link rel="stylesheet" href="../common/common.css">
+		<link rel="stylesheet" href="common/common.css">
```

Comment:
I doubt that works…

You can of course make the path an absolute one.

Reply:
But the path for the JS file does look like this `common/common.js`...

Reply:
Well I thought just make it explicitly relative (like`/commons/…`) then, but well… yeah. Just strange why it worked then in the past.

Well… basically does not really matter.